.github/workflows/tempo-ci.yaml

```diff
@@ -0,0 +1,41 @@
+name: Tempo Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  test-tempo:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the parquet-go repository
+      - name: Checkout parquet-go
+        uses: actions/checkout@v3
+
+      # Step 2: Clone the Tempo repository
+      - name: Clone Tempo
+        run: git clone https://github.com/grafana/tempo.git
+
+      # Step 3: Set up Go (if not already set up in the runner)
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'  # Use the appropriate Go version for Tempo
```

Comment:
Can we make this a matrix of Go versions that parquet-go supports? Otherwise, I'm concerned that we might be missing issues due to having code that target newer versions of Go.